chapters/io/index.rst

```diff
@@ -0,0 +1,315 @@
+How to handle input and output for Flows
+=======================================
+
+This chapter explains how to provide input data to the `Flow API`. Moreover, it describes how to handle the output via
```

Comment:
Why the line break in the sentence?

Reply:
I wanted to have less than 120 characters in one line. Now, I made the line break after the first sentence. One line break is ignored by the rendering anyways.

Reply:
We shouldn't break paragraphs https://jinaai.slack.com/archives/C018QCARHNG/p1612284457147700